pages/home.js

```diff
@@ -76,50 +80,73 @@ export default function Home(props) {
       })
   }
 
-  let onCreateHandler = (e) => {
+  let onCreateHandler = async (e) => {
     //prevent default behaviour of form
     e.preventDefault()
 
-    let valid = true
+    let username = owner.value,
+      userid = document.cookie.split('userid=')[1].substring(0, 5) || undefined
```

Comment:
Getting an error here calling substring on undefined when no userid cookie exists

Reply:
Added checks to see if cookie exists on both pages